spec/unit/shoulda/matchers/active_record/have_db_column_matcher_spec.rb

```diff
@@ -98,6 +98,14 @@
     end
   end
 
+  context 'with invalid argument option' do
+    it 'raises an error with the unknown options' do
+      expect {
+        have_db_column(:salary).with_options(preccision: 5, primaryy: true)
+      }.to raise_error("Unknown option(s): 'preccision, primaryy'")
```

Comment:
Okay, so now that you're testing multiple options, it seems like we could improve this message a bit. What if you change the above to 

``` ruby
raise ArgumentError, "Unknown option(s): #{invalid_options.map(&:inspect).join(", ")}"
```

Then you should get something like:

``` ruby
expect {
  have_db_column(:salary).with_options(preccision: 5, primaryy: true)
}.to raise_error("Unknown option(s): :preccision, :primaryy")
```

Reply:
Ok! I improved the message!